app/build.gradle

```diff
@@ -49,13 +49,13 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:$Versions.rxAndroid"
     implementation "com.jakewharton.rxbinding3:rxbinding:$Versions.rxBinding"
     implementation 'com.google.android.material:material:1.1.0-alpha03'
-    implementation "com.squareup.retrofit2:retrofit:$Versions.retrofit"
-    implementation "com.squareup.retrofit2:adapter-rxjava2:$Versions.retrofit"
-    implementation "com.squareup.retrofit2:converter-gson:$Versions.retrofit"
+//    implementation "com.squareup.retrofit2:retrofit:$Versions.retrofit"
+//    implementation "com.squareup.retrofit2:adapter-rxjava2:$Versions.retrofit"
+//    implementation "com.squareup.retrofit2:converter-gson:$Versions.retrofit"
     implementation "com.github.bumptech.glide:glide:$Versions.glide"
     kapt "com.github.bumptech.glide:compiler:$Versions.glide"
     implementation "com.facebook.stetho:stetho:$Versions.stetho"
-    implementation "com.facebook.stetho:stetho-okhttp3:$Versions.stetho"
+
     implementation "com.squareup.picasso:picasso:$Versions.picasso"
     implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0-alpha01'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/2276cbdbb6d20c028c5b71240ba140ca752f1231/app/build.gradle#L60"></span>A newer version of androidx.lifecycle:lifecycle-extensions than 2.1.0-alpha01 is available: 2.1.0-alpha02</td>
    </tr>
  </tbody>
</table>